### PR TITLE
Capstone

### PR DESCRIPTION
**Pull request by gedewon**
## Capstone PR
* Home Page, About page built-in responsive manner 
* Dynamically inject Data to HTML page from JS 
* Hamburger menu functionality 
* Deploy to git hub page 
* Loom Link https://www.loom.com/share/c10eb6fdb4f14facbe84fa930c6e12b2
* ADD readme to hold the acknowledgment to the Designer

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+          <div class="past-links">
+            <a href="#" target="_blank">
+              <img src="Images/about/about_2013.png" alt="CCSumit2013" />
+            </a>
+            <a href="#" target="_blank">
+              <img src="Images/about/about_2011.png" alt="CCSumit2011" />
+            </a>
+          </div>
+        </div>
+      </section>
```
Comment (nicupop729) [inline about.html:183]: The About section lacks the footer, as per the design provided.

- [ ] Kindly implement the footer for the about page for both mobile and desktop versions.

Note that the footer for mobile is different from the one for desktop.

The desktop version of the footer for the About page:
image

The mobile version of the footer for the About page:
image

File: index.html [hunk 2]
```diff
@@ -15,10 +15,249 @@
...
+                  <p>
+                    Create opportunities to freely network with CC personnel from
+                    around the world to ignite your ideas
+                  </p>
+                </div>
+              </div>
+            </div>
+            <a href="#" class="more-link">view full program </a>
+            <button class="join-button" type="button">
+              Join CC Global Summit 2015
```
Comment (nicupop729) [inline index.html:216]: Kindly adapt the design of this part of the project to look more closely as the provided one, with more height or more vertical padding and a better spacing in it's container, for the mobile version:

Your design for the Join Summit button:
image

Cindy Shin's design:
image

Comment (nicupop729) [reply index.html:216]: Regarding the desktop version, as well, please take a closer look at the provided design and try to match yours with it, maybe by applying a bit more margin-top to the link;

Your design:
image

Cindy Shin's design:
image

File: Styles/style.css [hunk 3]
```diff
@@ -0,0 +1,976 @@
+@media only screen and (max-width: 767px) {
```
Comment (nicupop729) [inline Styles/style.css:1]: Taking into account that we are building following the mobile-first approach regarding the style, this first media query is redundant;
We should build the style from mobile. And for the larger screen, we just adapt the style for the components that need to be changed. In this way, we don't violate the DRY (Don't Repeat Yourself) principle. 
 - [ ] you can take into account to remove this media query.
 - [ ] for the desktop, apply the change only for the elements that need to be changed;

File: Styles/style.css [hunk 4]
```diff
@@ -0,0 +1,976 @@
...
+    display: none;
+  }
+
+  body {
+    display: unset;
+    margin: 0;
+    padding: 0;
+    font-family: nanum-gothic, sans-serif;
+    color: rgb(51, 51, 51);
+  }
```
Comment (nicupop729) [inline Styles/style.css:550]: This part of the code is completely repeating from the beginning of this CSS file. if you remove that redundant media query for mobile, you won't have to repeat this code again. 

- [x] Kindly, take into consideration removing all the repeating code you have in this 2 media queries;

**Review by nicupop729 — CHANGES_REQUESTED** [review]
# STATUS: Required Changes ♻️

Hi @Gedewon, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### Some highlights for you
* Correct GitHub flow practice ✔️ 
* Good commit messages ✔️ 
* Professional README ✔️ 

### Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: about.html (hunk 1 above)
Comment (gedewon) [reply about.html:183]: Thank you for your feedback!! addressed.

File: index.html (hunk 2 above)
Comment (gedewon) [reply index.html:216]: Thank you !! resolved.

File: Styles/style.css (hunk 4 above)
Comment (gedewon) [reply Styles/style.css:550]: Thank you. scape plan with the media section up to 300px.

File: about.html (hunk 1 above)
Comment (OLIPLICHE) [reply about.html:183]: ## OPTIONAL
Please the desktop and mobile versions of the footer for the About page are not well implemented, kindly fix that

**Review by OLIPLICHE — APPROVED** [review]
Hi @Gedewon, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: about.html (hunk 1 above)
Comment (gedewon) [reply about.html:183]: Thank you for your Feedback !!! resolved.

**Pull request merged**